tests/js/arrow1.ts

```diff
@@ -34,6 +34,24 @@ test("read file", async (t) => {
   t.end();
 });
 
+test("read file async", async (t) => {
+  const server = await temporaryServer();
```

Comment:
Amazing; I'd wanted to have an async test for a while, ref https://github.com/kylebarron/parquet-wasm/pull/207

is it possible to test the new stream API too?